test.py

```diff
@@ -100,11 +100,26 @@ def test_update_ideal_temp_returns_response_200(self, mock_query, mock_db_sessio
             }
 
             response = self.client.patch('/destinations',
-                                         json = {'current_temperature': 23,
-                                         'destination_name': 'Test','new_temperature': 25})
+                                         json = {'destination_name': 'Test','new_temperature': 25})
 
             self.assertEqual(response.status_code, 200)
 
+    @patch("main.db.session")
+    @patch("main.Destinations.query")
+    def test_update_ideal_temp_with_missing_name_returns_response_500(self, mock_query, mock_db_session):
+        with app.app_context():
+            response = self.client.patch('/destinations',
+                                            json = {'destination_name': 'Test', 'new_temperature': 25})
+
+            self.assertEqual(response.status_code, 500)
```

Comment:
What are you testing here? Status cofe 500 is when we hit an exception? Is it missing the temperature variable? Because that seems to be in the JSON so its not missing